HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,714 @@
+
+This HIP changes this behavior in the following ways:
+
+- The value `-1` is now permitted for `max_auto_associations`, and this value means “unlimited”.
+- The account holder no longer prepays for unused slots regardless of the setting for `max_auto_associations`. Slots are only paid for when used, and are initially paid for by the sender during automatic associations. The account is responsible for covering rent for each association after the first auto-renewal period. `used_auto_associations` is incremented every time an auto-association is made.
+- The default value for automatically-created accounts will now be `-1`. This means, if an account is created automatically (by performing a token transfer to an alias that does not yet exist, see [HIP-583](https://hips.hedera.com/hip/hip-583)), then the automatically created account is configured with unlimited automatic token associations. Accounts created via the normal flow will continue to have a `used_auto_associations` default of 0.
+
+### Crypto Transfers
+
+Normal crypto-transfers remain the same as today, with one difference. If Account A sends a token to Account B, and Account B is not associated with that token but has an available auto-association slot, then Account A will pay for that association and the first auto-renewal period’s rent, in addition to the typical transfer costs. In summary previously the reciever would pay for an association in advance, not the payer of the transaction pays.
+
+![Crypto Transfer Transaction](../assets/hip-904/frictionless-airdrop-cryptoTransferTransaction.png)
+
+### Airdrops
+
+Airdrops are similar to crypto transfers, but differ in one fundamental way - when an airdrop is performed and the receiver does not have available or automatic association slots, rather than failing, the transfer will be kept in state as a *pending transfer*. To represent this new semantic, there will be a new `TokenAirdropTransaction` meant for use specifically by those attempting an airdrop, with the following semantics:
+
+![Token Airdrop Transaction](../assets/hip-904/frictionless-airdrop-tokenAirdropTransaction.png)
+
+> ⚠️ `TokenAirdropTransaction` will not support tokens with custom fallback fees that require the receiver to pay the fallback fee.
+
+
+When a sender airdrops a token, if the receiver’s account is already associated with the token, then it is effectively treated as a normal crypto transfer, except that an *airdrop fee* is assessed. This fee exists for all uses of `TokenAirdropTransaction` as a spam-deterrent fee.
+
```

Comment:
Is the airdrop fee noted here mean the transaction fee for a `TokenAirDropTransaction`?

if so, suggestion:


When a sender airdrops a token, if the receiver’s account is already associated with the token, then it is effectively treated as a normal crypto transfer, except that an *airdrop transaction fee* is assessed. This fee exists for all uses of `TokenAirdropTransaction` as a spam-deterrent fee.

If this is different fee in addition to a transaction fee for the TokenAirdropTransaction then we should define this in the terminology section. Suggestion:

When a sender airdrops a token, if the receiver’s account is already associated with the token, then it is effectively treated as a normal crypto transfer, except that an *airdrop fee* is assessed. This fee exists for all uses of `TokenAirdropTransaction` as a spam-deterrent fee. The airdrop fee is a fee in addition to a the transaction fee charged for the `TokenAirdropTransaction`.

Reply:
It's the former. Adopted your suggestion. Thanks